### ISSUE TITLE
What does self.seen mean in dataset.py?

### ISSUE THREAD

**Issue by gauss-clb** [marvis/pytorch-yolo2#53]
What does self.seen mean in [dataset.py](https://github.com/marvis/pytorch-yolo2/blob/master/dataset.py#L27)?

**Comment by JasonZhangHkust** [comment]
self.seen means how many pictures have been used for the training.

**Comment by gauss-clb** [comment]
@JasonZhangHkust Could you answer my other questions?

**Comment by JasonZhangHkust** [comment]
Hi, I think that is just for display. If not used in the training mode, the ``listDataset``  loads the boxes which are greater than the scale. While in training mode, the ``listDataset`` just loads all ground truth bounding boxes for training.

**Comment by omg777** [comment]
@JasonZhangHkust Using different data augmentation is contained in paper ?
